client/my-sites/post-type-list/post-action-counts/index.jsx

```diff
@@ -58,15 +75,24 @@ class PostActionCounts extends PureComponent {
 	}
 
 	renderLikeCount() {
-		const { likeCount: count, numberFormat, postId, showLikes, siteSlug, translate } = this.props;
+		const {
+			likeCount: count,
+			numberFormat,
+			siteId,
+			postId,
+			showLikes,
+			siteSlug,
+			translate,
+			isCurrentLikesPopoverOpen,
+		} = this.props;
 
 		if ( count < 1 || ! showLikes ) {
 			return null;
 		}
 
 		return (
-			<li>
-				<a href={ `/stats/post/${ postId }/${ siteSlug }` } onClick={ this.onActionClick( 'likes' ) } >
+			<li ref={ this.setLikesPopoverContext }>
+				<a href={ `/stats/post/${ postId }/${ siteSlug }` } onClick={ this.onLikesClick }>
```

Comment:
I think it's probably a good idea to preserve the `href` here - this retains the ability to right-click and open this link in a new tab.